package.json

```diff
@@ -7,15 +7,15 @@
     "node": ">=8.9.0 <9.0"
   },
   "scripts": {
-    "start": "cross-env NODE_PATH=. node server.js",
+    "start": "node server.js",
```

Comment:
Can we either use `cross-env` or remove it as a dependency? I'm not sure if we actually require it for Azure?

Reply:
Yeah, not sure what shell Azure uses, and whether cross-env is required for it.  I can try removing it and seeing if the tests still pass if you like?

Reply:
Yeah worth a shot 👍

Reply:
I guess the other thing is it should be left in for any devs working on Windows?

Reply:
Hmmm maybe